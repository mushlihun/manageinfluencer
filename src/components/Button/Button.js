import React from 'react'
import styled from 'styled-components/macro'
import PropTypes from 'prop-types'

Button.propTypes = {
  headline: PropTypes.string,
}

export default function Button({ headline }) {
  return <ButtonStyled>{headline}</ButtonStyled>
}

const ButtonStyled = styled.button`
  color: var(--main-font-color);
  font-family: var(--font-weight-light);
  background: var(--darkgreen);
  border: var(--darkgreen);
  box-shadow: 0 3px 10px var(--boxshadow-color);
  font-size: 18px;
  border-radius: 40px;
  width: 130px;
  height: 75px;
  display: block;
  border: none;
  margin-right: auto;
  margin-left: auto;
  margin-bottom: 60px;
  cursor: pointer;
`
