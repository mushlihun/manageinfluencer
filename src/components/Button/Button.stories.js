import React from 'react'
import Button from './Button'

export default {
  title: 'Components/Button',
  component: Button,
}

export const Contact = () => <Button headline="Contact" />
export const Submit = () => <Button headline="Submit" />
