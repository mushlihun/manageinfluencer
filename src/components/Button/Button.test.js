import React from 'react'
import renderer from 'react-test-renderer'
import Button from './Button'
import '@testing-library/jest-dom/extend-expect'

describe('Button', () => {
  it('renders Button correctly', () => {
    const tree = renderer.create(<Button />)
    expect(tree).toMatchSnapshot()
  })
})
