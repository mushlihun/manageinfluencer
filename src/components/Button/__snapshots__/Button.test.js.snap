// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders Button correctly 1`] = `
<button
  className="Button__ButtonStyled-kr25vp-0 hVIQhu"
/>
`;
